dissect/target/plugins/apps/webhosting/cpanel.py

```diff
@@ -0,0 +1,68 @@
+from datetime import datetime
+from typing import Iterator
+
+from dissect.target.helpers.record import TargetRecordDescriptor
+from dissect.target.plugin import Plugin, export
+
+CpanelLastloginRecord = TargetRecordDescriptor(
+    "application/log/cpanel/lastlogin",
+    [
+        ("datetime", "ts"),
+        ("string", "user"),
+        ("net.ipaddress", "remote_ip"),
+    ],
+)
+
+CPANEL_LASTLOGIN = ".lastlogin"
+CPANEL_LOGS_PATH = "/usr/local/cpanel/logs"
+
+
+class CpanelPlugin(Plugin):
+    # TODO: Parse other log files https://support.cartika.com/portal/en/kb/articles/whm-cpanel-log-files-and-locations
+    __namespace__ = "cpanel"
+
+    def check_compatible(self) -> None:
+        return bool(self.target.fs.path(CPANEL_LOGS_PATH).exists())
```

Comment:
What is preferred based on this request and the documentation (https://github.com/fox-it/dissect.target/blob/main/dissect/target/plugins/general/example.py#L61-L64)?
* Return `True` or `False`
* If `False` and `UnsupportedPluginError`
* Return `True` or `UnsupportedPluginError`

Reply:
It's very inconsistent right now, but I believe we intend to ditch the `bool` return path of `check_compatible` and let that be `raise UnsupportedPluginError` (or return `None` if compatible).

Either way the error message should say `cPanel` instead of `cpanel` :)